src/main/java/uk/gov/justice/digital/delius/controller/api/OffenderController.java

```diff
@@ -244,7 +260,7 @@ private DocumentMeta documentMetaOf(uk.gov.justice.digital.delius.data.api.alfre
     @JwtValidation
     public ResponseEntity<OffenderIdsResource> getOffenderIds(
             final @RequestHeader HttpHeaders httpHeaders,
-            final @RequestParam(name = "pageSize", required = false, defaultValue = "${offender.ids.pagesize:1000}") int pageSize,
+            @ApiParam(defaultValue = "1000") final @RequestParam(name = "pageSize", required = false, defaultValue = "${offender.ids.pagesize:1000}") int pageSize,
```

Comment:
I'm dubious whether this actually gets injected with the correct value, but have just changed the docs - was generating json parse errors as it was